datasets/us/cbp_forced_labor/crawler.py

```diff
@@ -0,0 +1,137 @@
+from zavod import Context, helpers as h
+from normality import slugify
+from typing import Dict, Generator, Any
+from lxml.html import HtmlElement
+import re
+
+pattern = r"\bFishing\s+Vessels\b"
+
+
+def crawl_item(context: Context, row: Dict[str, Any]):
+    try:
+        # Ensure all necessary keys are in the row
+        # Map the special case headers to expected names
+        if "#" in row:
+            row["id"] = row.pop("#")
+        # Extract required keys and handle them
+        country = row.pop("main_header")
+        internal_id = row.pop("id")
+        name = row.pop("entities")
+        name_result = context.lookup("name", name)
+        listing_date = h.parse_date(row.pop("date"), formats=["%m/%d/%Y"])
+        merchandise = row.pop("merchandise")
+        status = row.pop("status")
+        status_notes = row.pop("status-notes")
+        status_notes_link = row.pop("status_notes_link", None)
+
+        # print(f"Processing entity: {name}, ID: {internal_id}")
+
+        if re.search(pattern, country, re.IGNORECASE):
```

Comment:
Does this need to be a regex? Or can it just be a string comparison like `if "fishing vessels" in country.lower()`?

If it really needs to be a regex, Id name the pattern so it reads more clearly, and compile it at module-level
```suggestion

REGEX_VESSEL = re.compile(r"\bFishing\s+Vessels\b")

def ...
    ...
        if REGEX_VESSEL.search(country, re.IGNORECASE):
```

Reply:
I assume there're some whitespaces, it only worked with the regex, so I'd keep it